meta-lxatac-software/recipes-core/lxatac-profile/lxatac-profile/01-labgrid.sh

```diff
@@ -2,4 +2,4 @@
 
 # export labgrid environment to be used with labgrid-client on the lxatac
 source /etc/labgrid/environment
-export LG_CROSSBAR="ws://${LABGRID_COORDINATOR_IP:?}:${LABGRID_COORDINATOR_PORT:?}/ws"
+export LG_COORDINATOR="${LABGRID_COORDINATOR_IP:?}:${LABGRID_COORDINATOR_PORT:?}"
```

Comment:
Maybe move this change to a different PR?

It is not really oelint-adv-related.